### PR TITLE
Create audio content type

### PR DESCRIPTION
**Pull request by Tjocksmurf**
The old site is quite different to the new one when it comes to fields so a pure copy would be out of line.

I cloned the video content type, then I removed the video fields and added audio fields from the old site.

Happy to have a chat about it if you have any objections.

### CODE REVIEW

**Comment by Tjocksmurf** [comment]
[Create Audio content type](https://trello.com/c/vyL75GjP/183-create-audio-content-type)

**Review by Benjen — CHANGES_REQUESTED** [review]
@Tjocksmurf as part of this task could you:

- Create a sample audio content page.
- Create a visual regression test for audio content page using Cypress/Percy.

**Review by Benjen — CHANGES_REQUESTED** [review]
Styling of Audio content page is not correct.

File: config/default/core.extension.yml [hunk 1]
```diff
@@ -77,6 +77,7 @@ module:
   metatag: 0
   metatag_dc: 0
   metatag_open_graph: 0
+  migrate: 0
```
Comment (Benjen) [inline config/default/core.extension.yml]: Migrate module should not be enabled by default.

**Review by Benjen — CHANGES_REQUESTED** [review]

File: config/default/core.extension.yml (hunk 1 above)
Comment (Tjocksmurf) [reply config/default/core.extension.yml]: Done

Suggestion for addition to pre-commit-hook:

```
### Check core.extensions for modules that should be disabled
pattern='migrate\|sample\|dblog'
if grep $pattern  config/default/core.extension.yml >> /dev/null; then
    echo "Illegal modules detected as enabled. Please disable listed above"
    grep $pattern config/default/core.extension.yml
    exit 1
fi
### End check core.extensions for modules that should be disabled
```

**Pull request merged**